lib/inputs/rhel/static.ts

```diff
@@ -0,0 +1,27 @@
+import { ExtractAction, ExtractedLayers } from "../../extractor/types";
+import { streamToJson } from "../../stream-utils";
+
+export const getRedHatReposContentAction: ExtractAction = {
+  actionName: "redhat-content-manifests",
+  filePathMatches: (filePath) => isRedHatContentManifest(filePath),
+  callback: streamToJson,
+};
+export function getRedHatReposFromExtractedLayers(
+  extractedLayers: ExtractedLayers,
+) {
+  const reposByLayer = {};
+  for (const filePath in extractedLayers) {
+    if (isRedHatContentManifest(filePath)) {
+      const content = extractedLayers[filePath][
+        "redhat-content-manifests"
+      ] as any;
```

Comment:
Could this be undefined?

Reply:
Since it's using the same function to determine, I want to say no. But, I'm wrapping in a try catch as a better practice in case something goes horribly wrong there (malformed files etc). Thanks!